CHANGELOG.md

```diff
@@ -16,5 +16,5 @@ the [releases page](https://github.com/Consensys/teku/releases).
 - Added a hidden flag `--Xfork-choice-updated-always-send-payload-attributes` which would cause
 payload attributes to be calculated and sent with every fcU. This could be useful for builders
 consuming the `payload_attributes` SSE events.
-
+- Added `--stop-vc-when-validator-slashed` option to stop the Validator Client when any owned validator is slashed (requires a BN that publishes the `attester_slashing` and `proposer_slashing` SSE events)
```

Comment:
naming of cli arg inconsistent - `--stop-vc-when-validator-slashed-enabled`

Reply:
My bad, must be rebase issue. Fixed